src/features/notification/constant.ts

```diff
@@ -34,4 +34,10 @@ export const INFOBIP_SENDER_ID = process.env.INFOBIP_SENDER_ID_PATH
   : ''
 
 export const USER_NOTIFICATION_DELIVERY_METHOD =
-  process.env.USER_NOTIFICATION_DELIVERY_METHOD || 'sms'
+  process.env.USER_NOTIFICATION_DELIVERY_METHOD || 'email'
+
+export const SENDGRID_API_KEY = process.env.SENDGRID_API_KEY as string
+
+export const COUNTRY_LOGO_URL = process.env.COUNTRY_LOGO_URL as string
```

Comment:
I think it should be better for these to be string | undefined as otherwise we would just be lying to typescript. Because we know that these won't available to us all the time, so should handle that case as well